src/utils/pokemonApi.js

```diff
@@ -0,0 +1,11 @@
+export const fetchPokemonData = async () => {
+  try {
+    const response = await fetch("http://localhost:3001/api/pokemon");
+    if (!response.ok) throw new Error("Failed to fetch Pokémon data");
+    const data = await response.json();
+    return data;
```

Comment:
Please, import `checkResponse` from `auth.js` not to duplicate the `ok` checking